examples/java/src/main/java/org/apache/beam/examples/DebuggingWordCount.java

```diff
@@ -62,12 +62,14 @@
  * }
  * </pre>
  *
- * <p>To execute this pipeline using the Dataflow service and the additional logging discussed
- * below, specify pipeline configuration:
+ * <p>To change the runner, specify:
+ * <pre>{@code
+ *   --runner=YOUR_SELECTED_RUNNER
+ * }
+ * </pre>
+ *
+ * <p>To use the additional logging discussed below, specify:
```

Comment:
this is still Dataflow specific -- https://github.com/apache/incubator-beam/blob/0f072414b58c664e030810a4f15028e6081788bd/runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/options/DataflowWorkerLoggingOptionsTest.java